### PR TITLE
New accessible light & dark syntax highlighting themes

### PR DESCRIPTION
**Pull request by fbeeper**
Responding to @BasThomas call for help on accessibility on this site, I'm bringing a proposal for accessible themes for the syntax highlighting.

I've used the awesome work of [mpchadwick/pygments-high-contrast-stylesheets](https://github.com/mpchadwick/pygments-high-contrast-stylesheets/blob/gh-pages/UNLICENSE.txt) to provide accessible themes.

I've chosen Solarized Light (with the site's light bg) for the light theme, and Monokai Dark (with the site's dark bg) for the dark one. Without more fine tuning these seem to fit decently with the rest of the site.

I hope this is something you like!

### CODE REVIEW

**Comment by fbeeper** [comment]
I decided to add further changes to this PR: I rebuilt the variables of the color palette and redefined the colors to make all text contrasts accessible.

File: _includes/social_share.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% assign edit_url = base_edit_url | append: file_name %}
 
 <div class="row center sharing">
-   <small class="text-muted text-center">
```
Comment (jessesquires) [inline _includes/social_share.html:11]: not a huge fan of this change, visually.

what about using `text-secondary` ?

(same for the other usages)

**Comment by jessesquires** [comment]
> I decided to add further changes to this PR: I rebuilt the variables of the color palette and redefined the colors to make all text contrasts accessible.

This is great! 🎉 

Another good improvement, which would actually make this easier probably, is switching to Sass. (#524) But that's probably too much for this PR.

**Comment by fbeeper** [comment]
h1. Light Mode
h2. Article
h3. Before
swiftweekly_light_before
h3. After
swiftweekly_light_after

h1. Dark Mode
h2. Article
h3. Before
swiftweekly_dark_before
h3. After
swiftweekly_dark_after

**Comment by fbeeper** [comment]
I've broken down some details:

# Light Mode
## Header
### Before
swiftweekly_header_light_before
### After
swiftweekly_header_light_after
## Pager
### Before
swiftweekly_paging_light_before
### After
swiftweekly_paging_light_after
## Syntax
### Before
swiftweekly_syntax_light_before
### After
swiftweekly_syntax_light_after
# Dark Mode
## Header
### Before
swiftweekly_header_dark_before
### After
swiftweekly_header_dark_after
## Pager
### Before
swiftweekly_paging_dark_before
### After
swiftweekly_paging_dark_after
## Syntax
### Before
swiftweekly_syntax_dark_before
### After
swiftweekly_syntax_dark_after

File: _includes/social_share.html (hunk 1 above)
Comment (fbeeper) [reply _includes/social_share.html:11]: For our convenience, the screenshots for this change are in my last comment ("Pager" section). 

In terms of the new CSS color palette variables, this is currently using `--body-text color`. If I don't understand wrongly, you'd like it as `--secondary-text`? If so, I'd be happy to change that, of course :)

**Comment by fbeeper** [comment]
> [...] which would actually make this easier probably, is switching to Sass. (#524) But that's probably too much for this PR.

TBH, I have ZERO experience with Sass. So, I may not be the best candidate to make that contribution these days. 🙈

**Comment by jessesquires** [comment]
🎉

**Pull request merged**